02.windows/04.solving-problems/03.adguard-logs/docs.en.md

```diff
@@ -24,3 +24,17 @@ For analysis and diagnostic of different problems support service might need AdG
 4. **Important**: set the logging level back to **Standard**. **Debug** logging level will slow down the application.
 
 5. Send the archive to the AdGuard [support service](mailto:support@adguard.com) (support@adguard.com), describe the problem and don't forget to mention the time when you reproduced the issue.
+
+**Note**: sometimes support team members may ask you to send *trace* logs. Then you will need to do the following:
+
+1. Stop AdGuard.
+
+2. Open the console.
+
+3. Go to the folder C:\Program Files (x86)\Adguard>Adguard.exe /trace.
```

Comment:
with the command `cd C:\Program Files (x86)\Adguard` if you are using 64bit Windows, and `cd C:\Program Files\Adguard` in case if 32 bit one)